### DISCUSSION TITLE
After crash: SQLite Studio doesn't start (Mac)

### DISCUSSION THREAD

**Discussion by Auerhahn22** [pawelsalawa/sqlitestudio#4518] [General]
I had a crash and can't stark the app now anymore. I rebooted my Mac several times, dean I do? the app and reinstalled it... all the same: It seems to start and to close immediately. I believe, I should delete the settings, but in »/Library/Application Support« I can't find the good file.

What can I do?

MacBook Pro 16 (Intel), MacOS 12.6

**Comment by pawelsalawa** [comment]
Here's described location of the settings file: https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location

Let me know if that solved your problem.
